src/main/java/uk/gov/hmcts/darts/admin/test/TestSupportController.java

```diff
@@ -75,12 +79,22 @@ public void cleanUpDataAfterFunctionalTests() {
 
         removeDailyLists(session);
 
+        removeUserCourthousePermissions(session, courthouseTrash);
+
         removeCourtHouses(session);
 
         session.getTransaction().commit();
         session.close();
     }
 
+    private void removeUserCourthousePermissions(Session session,List cthIds) {
+        session.createNativeQuery("""
```

Comment:
No need for a multi-line string here?
I'm also not feeling it is right to simply remove the links in table security_group_courthouse_ae, unless we can be certain that it belongs only to a TEST security_group / courthouse?
I can't easily see this to know currently

Reply:
Don't actually need multi-line here - but it's consistent with the rest of the code in this class.